test/SourceKit/Indexing/index_objc_dynamic_refs.swift.response

```diff
@@ -0,0 +1,207 @@
+          key.line: 12,
+          key.column: 14,
+          key.entities: [
+            {
+              key.kind: source.lang.swift.decl.function.accessor.getter,
+              key.usr: "s:23index_objc_dynamic_refs6AClassC0B8DelegateAA9AProtocol_pSgvg",
+              key.line: 12,
+              key.column: 14,
+              key.is_dynamic: 1,
+              key.is_implicit: 1,
+              key.attributes: [
+                {
+                  key.attribute: source.decl.attribute.weak
+                }
+              ]
+            },
+            {
+              key.kind: source.lang.swift.decl.function.accessor.setter,
+              key.usr: "s:23index_objc_dynamic_refs6AClassC0B8DelegateAA9AProtocol_pSgvs",
+              key.line: 12,
+              key.column: 14,
+              key.is_dynamic: 1,
+              key.is_implicit: 1,
+              key.attributes: [
+                {
+                  key.attribute: source.decl.attribute.weak
+                }
+              ]
+            }
+          ],
+          key.attributes: [
+            {
+              key.attribute: source.decl.attribute.weak
+            }
+          ]
+        },
+        {
+          key.kind: source.lang.swift.ref.protocol,
+          key.name: "AProtocol",
+          key.usr: "c:@M@index_objc_dynamic_refs@objc(pl)AProtocol",
+          key.line: 12,
+          key.column: 28
+        },
+        {
+          key.kind: source.lang.swift.decl.function.method.instance,
+          key.name: "doSomething()",
+          key.usr: "s:23index_objc_dynamic_refs6AClassC11doSomethingyyF",
+          key.line: 14,
+          key.column: 10,
+          key.is_dynamic: 1,
+          key.entities: [
+            {
+              key.kind: source.lang.swift.ref.function.method.instance,
+              key.name: "dynamicMethod()",
+              key.usr: "c:@M@index_objc_dynamic_refs@objc(pl)AProtocol(im)dynamicMethod",
+              key.line: 15,
+              key.column: 23
+            },
+            {
+              key.kind: source.lang.swift.ref.var.instance,
+              key.name: "objcDelegate",
+              key.usr: "s:23index_objc_dynamic_refs6AClassC0B8DelegateAA9AProtocol_pSgvp",
+              key.line: 15,
+              key.column: 9,
+              key.entities: [
+                {
+                  key.kind: source.lang.swift.ref.function.accessor.getter,
+                  key.usr: "s:23index_objc_dynamic_refs6AClassC0B8DelegateAA9AProtocol_pSgvg",
+                  key.line: 15,
+                  key.column: 9,
+                  key.receiver_usr: "s:23index_objc_dynamic_refs6AClassC",
+                  key.is_dynamic: 1,
+                  key.is_implicit: 1
+                }
+              ]
+            },
+            {
+              key.kind: source.lang.swift.ref.var.instance,
+              key.name: "property",
+              key.usr: "s:23index_objc_dynamic_refs9AProtocolP8propertySSvp",
+              key.line: 16,
```

Comment:
Notice that the order is wrong (the reference is coming before the base expression). I'm not sure why that's the case...